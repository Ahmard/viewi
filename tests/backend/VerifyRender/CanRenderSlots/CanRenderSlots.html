<div>$title</div>
<CanRenderSlotsItem>This is slot content: $slotData</CanRenderSlotsItem>