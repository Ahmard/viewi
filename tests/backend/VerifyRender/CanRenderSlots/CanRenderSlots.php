<?php

class CanRenderSlotsComponent extends BaseComponent
{
    public string $title = 'Testing child component';
    public string $slotData = 'Slot data';
    function __construct()
    {
    }
}
