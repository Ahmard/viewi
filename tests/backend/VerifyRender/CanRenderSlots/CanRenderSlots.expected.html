<div>Testing child component</div>
<div>Slot part Title</div>
This is slot content: Slot data